examples/architecture_improvements_demo.py

```
@@ -0,0 +1,631 @@
+    demos = [
+        demo_command_queue_with_priorities,
+        demo_stream_processing,
+        demo_semantic_context,
+        demo_predictive_execution,
+        demo_integration_workflow,
+    ]
+
+    for demo in demos:
+        try:
+            await demo()
+        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)